.github/workflows/release.yml

```diff
@@ -179,17 +154,58 @@ jobs:
           GITHUB_REPOSITORY: ${{ github.repository }}
           GORELEASER_KEY: ${{ secrets.GORELEASER_KEY }}
 
+  docker-build-push:
```

Comment:
question: This job doesn't need the release or prepare at all? It just pushes?

Reply:
The docker container has everything it needs to run without being dependent on release or prepare 👍

Reply:
If the release or prepare fail then it will still push docker container?

Reply:
If the docker image can build, then that's fine with me. There will always be the possibility of one or the other failing regardless of the order or lack thereof.